client/app/js/controllers/main.js

```diff
@@ -549,7 +524,7 @@ controller('DisableEncryptionCtrl', ['$scope', '$uibModalInstance', function($sc
       $uibModalInstance.close(true);
     };
 }]).
-controller('IntroCtrl', ['$scope', '$rootScope', '$uibModalInstance', function ($scope, $rootScope, $uibModalInstance) {
+controller('IntroCtrl', ['$scope', '$rootScope', '$uibModalInstance', 'GLTranslate', function ($scope, $rootScope, $uibModalInstance, GLTranslate) {
```

Comment:
Yep, let's remove it as it is something used only in the amnesty project and never that integrated.

as you had to touch it in this branch and it is a removal ok for remove this here in this branch.